src/app/store/actions/VehicleTestResultModel.actions.spec.ts

```diff
@@ -3,7 +3,25 @@ import {
 } from './VehicleTestResultModel.actions';
 import {TestResultModel} from '@app/models/test-result.model';
 
-const testResultModelExample = {
+const testResultModelExample: TestResultModel = {
```

Comment:
Can we move this into utils/testing.utils.ts file please so we will reuse it for subsequent tests.
We could export a function that returns the testResultModel mock.